docs/Deploying-Pega-on-EKS.md

```diff
@@ -0,0 +1,578 @@
+
+1. Use a web browser to log in to <https://aws.amazon.com/console/>, which logs you into your default region.
+
+2. In the AWS Management Console, use the search tool to navigate to **Amazon RDS** main page.
+
+3. In **Amazon RDS** main page, in the Create Database section, click **Create database**.
+
+    This creates an RDS database in your default region. If this region is different from the region where your EKS worker nodes are located, you must reset your region to the region when you created your EKS cluster.
+
+4. In a new AWS console, create an Amazon RDS parameter group for this new database that it can use to limit the number of database connections.
+
+   a.  Open the Amazon RDS navigation (menu in upper left of screen) and right-click **Parameter groups** to open the menu in a new screen.
+
+   b. Click **Create parameter group**.
+
+   c. Enter the following group details:
+
+   - in **Parameter group family** select the database family to which this parameter group will apply; this family is based on the PostgreSQL engine and version you chose for your database instance.
+
+   - in **Type** select `DB parameter group`.
+
+   - in **Group name** enter an applicable name for this database parameter group.
+
+   - in **Description** enter any distinguishing details for this parameter group, such as `DB connection limitation settings`.
+
+   d. Click **Create**.
+
+   e. Select the new parameter group created and click on it to open.
+
+   f. In the search area, enter `max_connections`, select the parameter and click **Edit parameters**.
+
+   g. Update the value to a minimum of 5000 and then click **Save changes**.
+
+   With your changes saved, you can close the parameter group window in your browser.
+
+5. To create your database, return to the **Create database** page and select the following options:
+
+   a. For **Choose a database creation method**, select **Standard create**.
+
+   b. For **Engine Options**, select **PostgreSQL**.
+
+   c. For **Version**, select the latest available PostgreSQL version 11 series. For example, select `PostgreSQL 11.9-R1`.
+
+   d. For **Templates**, select **Production**.
+
+   e. In the **Settings** section, specify these details:
+
+   - In the **DB instance identifier**, enter a unique \<*databasename*\>.
+   - In **Credentials Settings**, leave **Master username** set to the default.
+   - Create a master password for your database that meets your organization standards and then confirm it.
+
+   f. In the **DB instance size** section, add these details:
+
+   - Select **Standard classes**.
+   - Select **db.m5.2xlarge** or greater. The **db.m5.2xlarge** selection provides the minimum hardware requirement for Pega Platform installations (a minimum of **4** vCores and **32 GB** storage).
+RAM
+   g. In the **Storage** section, for details, accept the default values:
+
+   - **Storage type** is **Provisioned IOPS**.
```

Comment:
Is provisoned iops necessary?  I think it's not usually used

Reply:
my current reviewers - NekhilK and MatteoC didn't call this out, so I am not sure --any final guidance would be useful!!